### PR TITLE
feat(gaussdb/nosql): add lb_ip_address and lb_port attributes

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=./huaweicloud/services/acceptance/gaussdb/ TESTARGS='-run=TestAccGaussInfluxInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb/ -v -run=TestAccGaussInfluxInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccGaussInfluxInstance_basic
=== PAUSE TestAccGaussInfluxInstance_basic
=== CONT  TestAccGaussInfluxInstance_basic
--- PASS: TestAccGaussInfluxInstance_basic (1535.34s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   1535.378s

make testacc TEST=./huaweicloud/services/acceptance/gaussdb/ TESTARGS='-run=TestAccGeminiDBInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb/ -v -run=TestAccGeminiDBInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccGeminiDBInstance_basic
=== PAUSE TestAccGeminiDBInstance_basic
=== CONT  TestAccGeminiDBInstance_basic
--- PASS: TestAccGeminiDBInstance_basic (1102.04s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   1102.073s

make testacc TEST=./huaweicloud/services/acceptance/gaussdb/ TESTARGS='-run=TestAccGaussMongoInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb/ -v -run=TestAccGaussMongoInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccGaussMongoInstance_basic
=== PAUSE TestAccGaussMongoInstance_basic
=== CONT  TestAccGaussMongoInstance_basic
--- PASS: TestAccGaussMongoInstance_basic (1269.21s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   1269.242s

make testacc TEST=./huaweicloud/services/acceptance/gaussdb/ TESTARGS='-run=TestAccGaussRedisInstance_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/gaussdb/ -v -run=TestAccGaussRedisInstance_basic -timeout 360m -parallel 4
=== RUN   TestAccGaussRedisInstance_basic
=== PAUSE TestAccGaussRedisInstance_basic
=== CONT  TestAccGaussRedisInstance_basic
--- PASS: TestAccGaussRedisInstance_basic (3324.12s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/gaussdb   3324.154s

```

### CODE REVIEW

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go [hunk 1]
```diff
@@ -487,7 +495,8 @@ func resourceGeminiDBInstanceV3Read(d *schema.ResourceData, meta interface{}) er
 	d.Set("security_group_id", instance.SecurityGroupId)
 	d.Set("dedicated_resource_id", instance.DedicatedResourceId)
 	d.Set("mode", instance.Mode)
-	d.Set("db_user_name", instance.DbUserName)
```
Comment (Lance52259) [inline huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go:490]: Cannot obtain DB user name from the API response?

Comment (niuzhenguo) [reply huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go:490]: that was a mistake.

File: huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go [hunk 2]
```diff
@@ -164,6 +164,14 @@ func ResourceGaussRedisInstanceV3() *schema.Resource {
...
 				Computed: true,
 			},
+			"lb_ip_address": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"lb_port": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go:174]: need to set these attributes in the resourceGaussRedisInstanceV3Read func

Comment (niuzhenguo) [reply huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go:174]: done

**Pull request merged**